src/mods/war.js

```diff
@@ -148,7 +148,7 @@ idrinth.war = {
             }
 
             cleanUp();
-            var tmpMagics = getMagic(data);
+            let tmpMagics = getMagic(data);
```

Comment:
In some cases, it's even better, performance wise, to use const instead of let ;) Only if your not planning to update/replace your variable anymore (const info - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const).